roles/run/files/schema.json

```diff
@@ -15,6 +15,9 @@
                 },
                 "name": {
                     "type": "string"
+                },
+                "requires_ansible": {
```

Comment:
@NilashishC I'm not really sure if I understand you. Do you want to have it _only_ valid for `vmware_rest`?

If yes, it might be better to implement `requires_ansible` directly in #76 instead of having this separate PR. And I'll have to find out how to implement this in JSON Schema...

Reply:
@NilashishC If I understood you correct, does [this](https://github.com/ansible-community/ansible.content_builder/pull/76/commits/0b08af08ae1f9b04a03ba4cf374144da9d5b04cb) look better to you? If yes, please close this comment and I'll concentrate on #76.

Reply:
@mariolenz That looks perfect! Thank you for addressing this. :)